src/components/Select/Select.tsx

```diff
@@ -110,11 +110,11 @@ export const Select: FC<SelectProps> = React.forwardRef(
     );
     const [dropdownVisible, setDropdownVisibility] = useState<boolean>(false);
     const [options, setOptions] = useState<SelectOption[]>(
-      _options.map((option: SelectOption, index: number) => ({
+      _options?.map((option: SelectOption, index: number) => ({
```

Comment:
do you want to fallback to an empty array so code can rely on this being an array?
```suggestion
      (_options || []).map((option: SelectOption, index: number) => ({
```